src/client/src/css/jobApplication.css

```diff
@@ -125,36 +125,13 @@
 }
 
 
-@media all and (max-width:560px) {
-    
-    .JAPContainer {
-        
-        width: 450px;
-        height: 1000px;
-    }
-
-    .JAPContainer .jobDetails {
-        
-        width: 75%;
-
-    }
-    
-    .JAPContainer .jobApplicationForm{
-
-        width: 75%;
-        display: flex;
-        flex-direction: column;
-        margin: none;
-        
-    }   
-}
 
 @media all and (max-width:560px) {
     
     .JAPContainer {
         
         width: 450px;
-        height: 1200px;
+        height: max-content;
```

Comment:
I made this gif to better explain what I mean. The footer is always there, in front of the user, even if a user scrolls to the bottom of the page. There's also some overlap with placeholders for some textboxes ![firefox_5nBhbv7c5o](https://user-images.githubusercontent.com/78051064/231321631-d4f054d9-df2c-4b17-915e-3dc6a353383b.gif)

You can also add a bottom-margin to the job application element.